pymongo/__init__.py

```diff
@@ -102,6 +102,9 @@
 from pymongo.read_preferences import ReadPreference
 from pymongo.write_concern import WriteConcern
 
+version = __version__
+"""Current version of PyMongo."""
+
```

Comment:
Another interesting difference is that after [PYTHON-1889](https://jira.mongodb.org/browse/PYTHON-1889) pymongo no longer includes `__annotations__` for these properties either:
```
>>> help(pymongo)
...
    __all__ = ['ASCENDING', 'DESCENDING', 'GEO2D', 'GEOSPHERE', 'HASHED', ...
    __annotations__ = {'__version__': <class 'str'>, 'version_tuple': typi...
    version = '4.3.1'
    version_tuple = (4, 3, 1)

VERSION
    4.3.1

```
vs
```
>>> help(pymongo)
...
    __all__ = ['ASCENDING', 'DESCENDING', 'GEO2D', 'GEOSPHERE', 'HASHED', ...
    version = '4.4.0.dev0'
    version_tuple = (4, 4, 0, '.dev0')

VERSION
    4.4.0.dev0
```

Any idea why that is?

Reply:
The annotations do not carry over across imports.  
Mypy still recognizes them:

```python
from pymongo import __version__, version, version_tuple

reveal_type(__version__)
reveal_type(version)
reveal_type(version_tuple)
```

```bash
mypy test.py
test.py:3: note: Revealed type is "builtins.str"
test.py:4: note: Revealed type is "builtins.str"
test.py:5: note: Revealed type is "builtins.tuple[Union[builtins.int, builtins.str], ...]"
Success: no issues found in 1 source file
```